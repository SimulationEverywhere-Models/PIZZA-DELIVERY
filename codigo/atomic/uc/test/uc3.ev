00:00:05:000 abrir_uc 1
00:00:08:000 lleno_p 1
00:00:12:000 recibe_ent_uc 1
00:00:18:000 lleno_ent_uc 1
00:00:22:000 recibe_ent_uc 2
00:00:30:000 lista_ent_uc 1
00:00:35:000 lista_p_uc 1
00:00:40:000 atiende_uc 5
00:00:53:000 cerrar_uc 1
00:00:55:000 recibe_ent_uc 5
