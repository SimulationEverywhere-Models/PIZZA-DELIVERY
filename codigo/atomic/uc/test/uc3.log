Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para uc(02)
Mensaje D / 00:00:00:000 / uc(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:05:000 / Root(00) / abrir_uc /      1.00000 para top(01)
Mensaje X / 00:00:05:000 / top(01) / abrir_uc /      1.00000 para uc(02)
Mensaje D / 00:00:05:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:05:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:05:000 / Root(00) para top(01)
Mensaje * / 00:00:05:000 / top(01) para uc(02)
Mensaje Y / 00:00:05:000 / uc(02) / lista_ll_uc /      1.00000 para top(01)
Mensaje D / 00:00:05:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:05:000 / top(01) / lista_ll_uc /      1.00000 para Root(00)
Mensaje D / 00:00:05:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:08:000 / Root(00) / lleno_p /      1.00000 para top(01)
Mensaje X / 00:00:08:000 / top(01) / lleno_p /      1.00000 para uc(02)
Mensaje D / 00:00:08:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:08:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:08:000 / Root(00) para top(01)
Mensaje * / 00:00:08:000 / top(01) para uc(02)
Mensaje Y / 00:00:08:000 / uc(02) / cerrado_uc /      1.00000 para top(01)
Mensaje D / 00:00:08:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:08:000 / top(01) / cerrado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:08:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:12:000 / Root(00) / recibe_ent_uc /      1.00000 para top(01)
Mensaje X / 00:00:12:000 / top(01) / recibe_ent_uc /      1.00000 para uc(02)
Mensaje D / 00:00:12:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:12:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:12:000 / Root(00) para top(01)
Mensaje * / 00:00:12:000 / top(01) para uc(02)
Mensaje Y / 00:00:12:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:00:12:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:12:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:00:12:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:12:030 / Root(00) para top(01)
Mensaje * / 00:00:12:030 / top(01) para uc(02)
Mensaje Y / 00:00:12:030 / uc(02) / dist_out_uc /      1.00000 para top(01)
Mensaje D / 00:00:12:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:12:030 / top(01) / dist_out_uc /      1.00000 para Root(00)
Mensaje D / 00:00:12:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:12:030 / Root(00) para top(01)
Mensaje * / 00:00:12:030 / top(01) para uc(02)
Mensaje Y / 00:00:12:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:00:12:030 / uc(02) / ... para top(01)
Mensaje Y / 00:00:12:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:00:12:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:18:000 / Root(00) / lleno_ent_uc /      1.00000 para top(01)
Mensaje X / 00:00:18:000 / top(01) / lleno_ent_uc /      1.00000 para uc(02)
Mensaje D / 00:00:18:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:18:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:18:000 / Root(00) para top(01)
Mensaje * / 00:00:18:000 / top(01) para uc(02)
Mensaje D / 00:00:18:000 / uc(02) / ... para top(01)
Mensaje D / 00:00:18:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:22:000 / Root(00) / recibe_ent_uc /      2.00000 para top(01)
Mensaje X / 00:00:22:000 / top(01) / recibe_ent_uc /      2.00000 para uc(02)
Mensaje D / 00:00:22:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:22:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:22:000 / Root(00) para top(01)
Mensaje * / 00:00:22:000 / top(01) para uc(02)
Mensaje Y / 00:00:22:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:00:22:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:22:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:00:22:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:22:030 / Root(00) para top(01)
Mensaje * / 00:00:22:030 / top(01) para uc(02)
Mensaje Y / 00:00:22:030 / uc(02) / dist_out_uc /      2.00000 para top(01)
Mensaje D / 00:00:22:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:22:030 / top(01) / dist_out_uc /      2.00000 para Root(00)
Mensaje D / 00:00:22:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:22:030 / Root(00) para top(01)
Mensaje * / 00:00:22:030 / top(01) para uc(02)
Mensaje Y / 00:00:22:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:00:22:030 / uc(02) / ... para top(01)
Mensaje Y / 00:00:22:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:00:22:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:30:000 / Root(00) / lista_ent_uc /      1.00000 para top(01)
Mensaje X / 00:00:30:000 / top(01) / lista_ent_uc /      1.00000 para uc(02)
Mensaje D / 00:00:30:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:30:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:30:000 / Root(00) para top(01)
Mensaje * / 00:00:30:000 / top(01) para uc(02)
Mensaje D / 00:00:30:000 / uc(02) / ... para top(01)
Mensaje D / 00:00:30:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:35:000 / Root(00) / lista_p_uc /      1.00000 para top(01)
Mensaje X / 00:00:35:000 / top(01) / lista_p_uc /      1.00000 para uc(02)
Mensaje D / 00:00:35:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:35:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:35:000 / Root(00) para top(01)
Mensaje * / 00:00:35:000 / top(01) para uc(02)
Mensaje Y / 00:00:35:000 / uc(02) / lista_ll_uc /      1.00000 para top(01)
Mensaje D / 00:00:35:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:35:000 / top(01) / lista_ll_uc /      1.00000 para Root(00)
Mensaje D / 00:00:35:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:40:000 / Root(00) / atiende_uc /      5.00000 para top(01)
Mensaje X / 00:00:40:000 / top(01) / atiende_uc /      5.00000 para uc(02)
Mensaje D / 00:00:40:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:40:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:40:000 / Root(00) para top(01)
Mensaje * / 00:00:40:000 / top(01) para uc(02)
Mensaje Y / 00:00:40:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:00:40:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:00:40:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:40:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:42:000 / Root(00) para top(01)
Mensaje * / 00:00:42:000 / top(01) para uc(02)
Mensaje Y / 00:00:42:000 / uc(02) / encolar_p /      5.00000 para top(01)
Mensaje D / 00:00:42:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:42:000 / top(01) / encolar_p /      5.00000 para Root(00)
Mensaje D / 00:00:42:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:42:000 / Root(00) para top(01)
Mensaje * / 00:00:42:000 / top(01) para uc(02)
Mensaje Y / 00:00:42:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:00:42:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:42:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:00:42:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:53:000 / Root(00) / cerrar_uc /      1.00000 para top(01)
Mensaje X / 00:00:53:000 / top(01) / cerrar_uc /      1.00000 para uc(02)
Mensaje D / 00:00:53:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:53:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:53:000 / Root(00) para top(01)
Mensaje * / 00:00:53:000 / top(01) para uc(02)
Mensaje Y / 00:00:53:000 / uc(02) / cerrado_uc /      1.00000 para top(01)
Mensaje D / 00:00:53:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:53:000 / top(01) / cerrado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:53:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:55:000 / Root(00) / recibe_ent_uc /      5.00000 para top(01)
Mensaje X / 00:00:55:000 / top(01) / recibe_ent_uc /      5.00000 para uc(02)
Mensaje D / 00:00:55:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:55:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:55:000 / Root(00) para top(01)
Mensaje * / 00:00:55:000 / top(01) para uc(02)
Mensaje Y / 00:00:55:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:00:55:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:55:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:00:55:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:55:030 / Root(00) para top(01)
Mensaje * / 00:00:55:030 / top(01) para uc(02)
Mensaje Y / 00:00:55:030 / uc(02) / dist_out_uc /      5.00000 para top(01)
Mensaje D / 00:00:55:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:55:030 / top(01) / dist_out_uc /      5.00000 para Root(00)
Mensaje D / 00:00:55:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:55:030 / Root(00) para top(01)
Mensaje * / 00:00:55:030 / top(01) para uc(02)
Mensaje Y / 00:00:55:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:00:55:030 / uc(02) / ... para top(01)
Mensaje Y / 00:00:55:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:00:55:030 / top(01) / ... para Root(00)
