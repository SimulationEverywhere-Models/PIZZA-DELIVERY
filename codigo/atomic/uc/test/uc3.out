00:00:05:000 lista_ll_uc 1
00:00:08:000 cerrado_uc 1
00:00:12:000 ocupada_ent_uc 1
00:00:12:030 dist_out_uc 1
00:00:12:030 libre_uc_ent 1
00:00:22:000 ocupada_ent_uc 1
00:00:22:030 dist_out_uc 2
00:00:22:030 libre_uc_ent 1
00:00:35:000 lista_ll_uc 1
00:00:40:000 ocupado_uc 1
00:00:42:000 encolar_p 5
00:00:42:000 libre_uc 1
00:00:53:000 cerrado_uc 1
00:00:55:000 ocupada_ent_uc 1
00:00:55:030 dist_out_uc 5
00:00:55:030 libre_uc_ent 1
